src/css/components/Breadcrumbs.scss

```diff
@@ -0,0 +1,33 @@
+@import "../config/index.scss";
+
+// * @define default Breadcrumbs
+
+.Nav-breadcrumb {
+  color: black;
+  font-size: 16px;
```

Comment:
There is a grid system in place that we can use here instead. I think this should be roughly `scaleGrid(2)` instead of `16px`.

Reply:
Done!